family2family/src/main/java/org/cbtf/f2f/model/Diagnosis.java

```diff
@@ -23,10 +23,18 @@
 
     private String description;
 
+    public Diagnosis(String description) {
```

Comment:
http://localhost:8080/internal-api/Diagnosis errors. Complains about not having a default constructor.